Makefile

```diff
@@ -16,7 +16,11 @@
 minimal: venv
 
 venv: requirements-dev.txt setup.py tox.ini
+ifeq ($(findstring .yelpcorp.com,$(shell hostname -f)), .yelpcorp.com)
+	tox -e venv-internal
+else
 	tox -e venv
+endif
```

Comment:
we technically don't need this anymore since we're no longer making a method that returns both slo and rollback files, but this seems small/useful enough that i didn't break it into its own PR (although i can if we want)